src/app/pages/your-virtrolio/your-virtrolio.component.ts

```diff
@@ -45,6 +57,14 @@ export class YourVirtrolioComponent implements OnInit {
     this.copyButtonText = 'Copied!';
   }
 
+  /**
+   * Closes sharing link modal and opens up desktop sharing modal
+   */
+  openDesktopSharing() {
+    $('#link-gen').modal('hide');
+    $('#share-link-modal').modal('show');
```

Comment:
Remove the word 'modal' from the ID. If you don't JQuery will act up (as I mentioned before);